docs/src/visualize.md

```diff
@@ -0,0 +1,17 @@
+# Visualize output
+
+In the examples we use [Makie.jl](https://makie.juliaplots.org/stable/) for plotting.
+
+Makie comes with few [backends](https://makie.juliaplots.org/stable/#makie_ecosystem). In the documented examples
+we use [CairoMakie](https://makie.juliaplots.org/stable/documentation/backends/cairomakie/) since this backend
+works well on headless devices, that is, devices without monitor. Since the documentation is automatically
+build via GitHub actions the CairoMakie backend is necessary. Users that run GeophysicalFlows.jl on
+devices with a monitor they might want to change to [GLMakie](https://makie.juliaplots.org/stable/documentation/backends/glmakie/)
+that display figures in an interactive window.
```

Comment:
which displays figures in an interactive window.

Reply:
are you sure it's "which" here? I thought it should be "that"... but I'm not a native English speaker...

Reply:
Yes now I look at it again my suggestions is perhaps not that good. Maybe clearest is "...change to the GLMakie backend that displays figures in an interactive window.

Reply:
Or maybe just display -> displays? Likely have overthought it!!